cloud/aws/bin/deploy.py

```diff
@@ -0,0 +1,26 @@
+#! /usr/bin/env python3
+import subprocess
+import os
+import sys
+
+sys.path.append(os.getcwd())
```

Comment:
Can this be modified by an upstream caller?

Reply:
Right now this is the first python script that will get run. we should move that into deploy-specific deploy script once we have those. Added a todo.

Reply:
Sounds good. One last request for posterity. Mind also adding a brief comment as to why it's necessary to do this in the middle of import statements?

Potential example:
```
# We modify the system path here so that the imports below can be resolved.
```

Reply:
Sounds good. One last request for posterity. Mind also adding a brief comment as to why it's necessary to do this in the middle of import statements?

Potential example:
```
# We modify the system path here so that the imports below can be resolved.
```